packages/compass-components/src/components/options-dropdown-button.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from 'react';
+import { Link, Icon } from './leafygreen';
+import { css } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const optionsButtonStyles = css({
+  backgroundColor: 'transparent',
+  border: 'none',
+  display: 'inline',
+  height: spacing[4] + spacing[1],
```

Comment:
I think combining spacings should be an exception, not the rule, this is basically a way to break from leafygreen spacings while making it look like we are using it. We do this in some cases, but is it really required here? Also does this component needs a hardcoded height? Would it make sense to just have some padding here instead?

Reply:
Probably don't need height here, going to remove. I think we'll need a width though so that the component doesn't move elements around it when the label changes. I'll update with that as well.